### ISSUE TITLE
package lock

### ISSUE THREAD

**Issue by vsoch** [akeshavan/neurobrain#3]
Hey @akeshavan I will likely have a bunch of noob questions, thanks in advance for teaching me :)

I wanted to ask about this warning:

```bash
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```
Is this the result of mixing yarn with... something else? Can be safely ignored? What generated the package-lock.json? I'm guessing it's good practice to include it with repos, but why is the suggestion here that it should be ignored or removed?

**Comment by vsoch** [comment]
omg the messages are color coded! I think I'm in :heart:

**Comment by akeshavan** [comment]
yes! At some point I accidentally ran `npm install` instead of `yarn install` and that really messes things up. I'm not sure how to get rid of that message, but it doesn't seem to break anything now?

also, i'm a noob too, this is my first npm package! haha
